java/src/jmri/jmrit/operations/trains/TrainManifestHeaderText.java

```diff
@@ -283,7 +283,7 @@ public static Element store() {
             e.addContent(values = new Element(Xml.OWNER));
             values.setAttribute(Xml.TEXT, getStringHeader_Owner());
         }
-        if (!getStringHeader_RWE().equals(Bundle.getMessage("RWE"))) {
+        if (!getStringHeader_RWE().equals(Bundle.getMessage("RWELabel"))) {
```

Comment:
As I read it, this displays "RWE:" in a line on the train manifest. Picked up from the ops.trains bundle. Makes the switch list more readable (IMHO). Changing the key name explains the difference from just plain "RWE" as used in the Set car RWE border name etc. (picked up from the common ops bundle)